dist/sri.json

```diff
@@ -1,20 +1,20 @@
 {
   "@dist/raven.js": {
     "hashes": {
-      "sha256": "YUHjJPYdEw3bT4F98xyt3oGoHXw900KdF7IyruSaLnA=",
-      "sha512": "i+iP/jcNGwly0KidFvgUcEAH4zzyeiWJIwWYVqRsUZaC2DH+U5O5YhVUOO6D1MZOBxGiqhCaCe4BxoQDOItnjQ=="
+      "sha256": "3GzWiT3dKqk4+iglDepSWh0Jbpa5exp52mS3maxUbYc=",
+      "sha512": "Gc0hZdIrJ5IW0a3IB+0mZupn0Kw35tUcMiS9tHKEmt97sSuqO8isQYM6Z6Lu0wSvkffTWUC/ZwpWpJlttnoa2A=="
     },
     "type": null,
-    "integrity": "sha256-YUHjJPYdEw3bT4F98xyt3oGoHXw900KdF7IyruSaLnA= sha512-i+iP/jcNGwly0KidFvgUcEAH4zzyeiWJIwWYVqRsUZaC2DH+U5O5YhVUOO6D1MZOBxGiqhCaCe4BxoQDOItnjQ==",
+    "integrity": "sha256-3GzWiT3dKqk4+iglDepSWh0Jbpa5exp52mS3maxUbYc= sha512-Gc0hZdIrJ5IW0a3IB+0mZupn0Kw35tUcMiS9tHKEmt97sSuqO8isQYM6Z6Lu0wSvkffTWUC/ZwpWpJlttnoa2A==",
     "path": "dist/raven.js"
   },
   "@dist/raven.min.js": {
     "hashes": {
-      "sha256": "iQZ79i+I68M6WGC5/+LDjAE+FgqzklvfzX4b3X0rA1k=",
-      "sha512": "U4qerTxhgVitu6fQ9YhZ+QpTlYhSWduvPKzQFBEwuxPNYNU/IP2RWdRrBte5lLFBZz7oD4yQVrXbaSOovZ+TSQ=="
+      "sha256": "0eJIUDDaN+Fxey9DBqD3xBRyGQlSiGuwKDWBU65zTyc=",
+      "sha512": "WEF1nEdHcnFHauBWTGQxZDC6gbJiJ3zQt8ddE+RajEbRM2H+yQRj/CYhwQMJ/DWQyRpzm+6ewCSiWel5/9RojQ=="
     },
     "type": null,
-    "integrity": "sha256-iQZ79i+I68M6WGC5/+LDjAE+FgqzklvfzX4b3X0rA1k= sha512-U4qerTxhgVitu6fQ9YhZ+QpTlYhSWduvPKzQFBEwuxPNYNU/IP2RWdRrBte5lLFBZz7oD4yQVrXbaSOovZ+TSQ==",
+    "integrity": "sha256-0eJIUDDaN+Fxey9DBqD3xBRyGQlSiGuwKDWBU65zTyc= sha512-WEF1nEdHcnFHauBWTGQxZDC6gbJiJ3zQt8ddE+RajEbRM2H+yQRj/CYhwQMJ/DWQyRpzm+6ewCSiWel5/9RojQ==",
     "path": "dist/raven.min.js"
   }
 }
```

Comment:
Ideally, I guess we'd want to reflect the new dist builds in here as well, but not a blocker by any means.